packages/blade/src/components/Checkbox/CheckboxGroup/CheckboxGroupLabel.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import getPlatformType from '../../../utils/getPlatformType';
+import BaseText from '../../Typography/BaseText';
+import { useCheckboxGroupContext } from './CheckboxGroupContext';
+
+type CheckboxGroupLabelProps = {
+  children: React.ReactNode;
+  id: string;
+};
+
+const CheckboxGroupLabel = ({ children, id }: CheckboxGroupLabelProps): React.ReactElement => {
+  const { labelPosition } = useCheckboxGroupContext();
+  const isReactNative = getPlatformType() === 'react-native';
+
+  const textNode = (
+    <BaseText
+      lineHeight="s"
+      fontFamily="text"
+      fontWeight="bold"
+      color="surface.text.subtle.lowContrast"
+      fontSize={75}
+    >
+      {children}
+    </BaseText>
+  );
+
+  // What harm can it do?
+  if (isReactNative) {
+    return textNode;
+  }
+
+  return (
+    <span style={{ width: labelPosition === 'left' ? '120px' : 'auto' }} id={id}>
```

Comment:
120px is the width of the form header. We do not have token for this. 

<img width="884" alt="Screenshot 2022-07-21 at 12 14 10 PM" src="https://user-images.githubusercontent.com/35374649/180147353-58aecab8-edb3-4ea2-82f0-c53e67f9ae95.png">